packages/sdk/src/internal/pagination.ts

```
@@ -0,0 +1,123 @@
+        if (!direction) {
+            this.direction = "Forward";
+        }
+    }
+
+    /**
+     * Creates a Pagination instance from a ToriiQueryBuilder
+     *
+     * @param query - The query builder to extract pagination parameters from
+     * @returns A new Pagination instance configured with the query's pagination settings
+     */
+    static fromQuery<T extends SchemaType, Inner extends any[]>(
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)